go/svc/graphql/pkg/subscription/subscription.go

```diff
@@ -0,0 +1,82 @@
+package subscription
+
+import (
+	"fmt"
+
+	"context"
+
+	"github.com/gomodule/redigo/redis"
+	"github.com/google/uuid"
+	"github.com/ispec-inc/monorepo/go/pkg/msgbs"
+	"github.com/ispec-inc/monorepo/go/svc/admin/pkg/redisbs"
+)
+
+var subscribers = make(map[msgbs.Event][]string)
```

Comment:
In the last implementation, we defined it once as var, but I think it would be better to define it as struct.
I thought it would be better to define it as a struct, and to have methods like subscribe and addSubscriber growing on this struct, so that the structure and behavior of the data would be coherent with the struct.